src/contracts/libraries/SlashingLib.sol

```diff
@@ -47,6 +47,15 @@ library SlashingLib {
         return x.mulDiv(WAD, y);
     }
 
+    /**
+     * @notice Used explicitly for calculating slashed magnitude, we want to ensure even in the
+     * situation where an operator is slashed several times and precision has been lost over time,
+     * an incoming slashing request isn't rounded down to 0 and an operator is able to avoided slashing penalties.
```

Comment:
nit: able to avoid

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/842/commits/ba893331d545b5feb09573287e5f37b1fc63a27c